src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -54,6 +54,22 @@ SetInteriorVehicleDataRequest::SetInteriorVehicleDataRequest(
 
 SetInteriorVehicleDataRequest::~SetInteriorVehicleDataRequest() {}
 
```

Comment:
Same comment as above.  this should be HMI's responsibility. Why are we changing timeout in SDL?

Reply:
@atiwari9
SDL should understand from its side whether the mobile request expired (and respond with `GENERIC_ERROR`) or HMI is still working on it. The reason for changing the timeout is the same as in the previous comment.

Reply:
https://github.com/smartdevicelink/sdl_core/pull/3726#discussion_r658758488